api/game_configurations/conditionals.yaml

```diff
@@ -11,43 +11,83 @@ games:
     prompt: placebolder
     level_number: 0
     inputs: int
-    num_inputs: 10
+    num_inputs: 2
+    prompt: >
+      You can practice basic (if) condition statement with basic blocks.
+      <br>
+      Yow will be given two integers (a, b), print either 'greater' or 'less' based on if condition
+      <ul>
+        <li>The <strong>Inputs</strong> will give you two integers randomly generated.</li>
+        <li>You need to use one of condition blocks from the category <strong>condition (if-else) </strong> </li>
+        <li>Instruction</li>
+        <ol>The <strong>logic_compare</strong> block can be plugged into input of <strong>if</strong></ol>
+        <ol>The <strong>output</strong> block can be plugged into input of <strong>do</strong> or <strong>else</strong></ol>
+        <ol>The <strong>text</strong> block can be plugged into input of the <strong>output</strong></ol>
+        <ol>You need to put 'greater' in the <strong>do</strong> & 'less' in the <strong>else</strong> </ol>
+      </ul>
     blocks:
-      - text
+      - input
       - output
+      - condition
+      - text
+      - math_number
+      - logic_compare
   - external_id: else
     title: What else is there?
     description: Throwing else into the mix
     prompt: placebolder
     status: active
     level_number: 1
     inputs: int
-    num_inputs: 10
+    num_inputs: 2
+    prompt: >
+      You can practice another version of condition (if /else) with boolean value [True/False]
```

Comment:
Can you rewrite this prompt? It's not clear to me what you're asking for here.

Reply:
Updated.